tests/test_reset.py

```diff
@@ -0,0 +1,154 @@
+    """
+    Contains vaious functions for interacting with the ADRL unreal environment
+    """
+
+    def __init__(self, drone_name="drone_1"):
+        """
+        Initializes class variables
+
+        Args:
+        -----
+            drone_name (string) = Name of user controlled drone (defaults to 'drone_1')
+        """
+        self.airsim_client = airsimdroneracinglab.MultirotorClient()
+        self.airsim_client_2 = airsimdroneracinglab.MultirotorClient()
+        self.airsim_client_3 = airsimdroneracinglab.MultirotorClient()
+        self.drone_name = drone_name
+        self.is_thread_active = False
+        self.thread_reset = threading.Thread(
+            target=self.repeat_timer, args=(self.reset, 0.05)
+        )
+        self.thread_reset_race = threading.Thread(
+            target=self.repeat_timer, args=(self.reset_race, 0.03)
+        )
+        self.thread_reset_and_reset_race = threading.Thread(
+            target=self.repeat_timer, args=(self.reset_and_reset_race, 0.09)
+        )
+        self.is_thread_active = False
+
+    def repeat_timer(self, callback, period):
+        """
+        Simple sleep timer
+        
+        Args:
+        -----
+            callback (function) = Function to call
+            period (float) = Repeat interval in seconds
+        """
+        while self.is_thread_active:
+            callback()
+            time.sleep(period)
+
+    def load_level(self, level_name, sleep_sec=2.0):
+        """
+        Loads given simulator level.
+
+        Args:
+        -----
+            level_name (string): Soccer_Field_Easy, Soccer_Field_Medium, ZhangJiaJie_Medium,
+                Building99_Hard, Qualification_Tier_1, Qualification_Tier_2, 
+                Qualification_Tier_3, Final_Tier_1, Final_Tier_2, or Final_Tier_3
+            sleep_sec (float, optional): Sleep time for loading level. Defaults to 2.0.
+        """
+        self.level_name = level_name
+        self.airsim_client.simLoadLevel(self.level_name)
+        self.airsim_client.confirmConnection()  # failsafe
+        time.sleep(sleep_sec)  # let the environment load completely
+
+    def reset(self):
+        """Resets Airsim cleint."""
+        print(time.time(), "called reset")
+        self.airsim_client.reset()
+
+    def reset_race(self):
+        """Resets current race."""
+        print(time.time(), "called simResetRace")
+        self.airsim_client_2.simResetRace()
+
+    def reset_and_reset_race(self):
+        """Resets Airsim cleint and current race"""
+        print(time.time(), "called reset, followed by simResetRace")
+        self.airsim_client_3.reset()
+        self.airsim_client_3.simResetRace()
+
+    def start_race(self, tier=1):
+        """
+        Starts race against baseline drone
+
+        Args:
+        -----
+            tier: (int, optional): Race tier determining difficulty. 
+                Tier 1: Planning only; Tier 2: Perception Only; Tier 3: Planning and Perception.
+        """
```

Comment:
Docstring fix? Ain't no way
```suggestion
    def start_race(self, tier: int | Optional=1):
        """
        Starts race against baseline drone

        Parameters
        ------------
            tier: int | Optional 
                Race tier determining difficulty. 
                Tier 1: Planning only; Tier 2: Perception Only; Tier 3: Planning and Perception.
        """
```